src/components/loginButton.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
```

Comment:
Let's also add some consistency to our naming conventions for the client now. I would go for LoginButton.tsx (PascalCase) instead.

Reply:
I renamed the file in VSCode but it only updated the import statement in Login.tsx, not the file itself.

Reply:
I was able to rename the file - I pushed the commit and it looks good now.

Reply:
Okay thanks!